components/src/components/SimpleContent/Component.ts

```diff
@@ -9,26 +9,43 @@ const ID = 'simplecontent';
 const DISPLAY = 'Static Text';
 
 export default class Component extends (ParentComponent as any) {
-    static schema(...extend) {
-        return ParentComponent.schema({
-            type: ID,
-            label: DISPLAY,
-            key: ID,
-            input: false,
-            html: ''
-        }, ...extend);
-    }
+  static schema(...extend) {
+    return ParentComponent.schema({
+      type: ID,
+      label: DISPLAY,
+      key: ID,
+      input: false,
+      html: ''
+    }, ...extend);
+  }
+
+  public static editForm = editForm;
+
+  static get builderInfo() {
+    return {
+      title: DISPLAY,
+      group: 'simple',
+      icon: 'pencil-square-o',
+      weight: 40,
+      documentation: Constants.DEFAULT_HELP_LINK,
+      schema: Component.schema()
+    };
+  }
 
-    public static editForm = editForm;
-
-    static get builderInfo() {
-        return {
-            title: DISPLAY,
-            group: 'simple',
-            icon: 'pencil-square-o',
-            weight: 40,
-            documentation: Constants.DEFAULT_HELP_LINK,
-            schema: Component.schema()
-        };
+  get defaultSchema() {
+    return Component.schema();
+  }
+
+  addTargetBlankToLinks = (html) => {
+    let result = '';
+    if (html.includes('<a href=')) {
+      result = html.replace(/<a href=/g, '<a target="_blank" href=');
     }
```

Comment:
i noticed some unexpected behavior when testing.
Entering just plain text (paragraph) into the wysiwyg editor didn't show up.
Also, saving the component and then going back to edit didn't work.

i think you need to return the original html in an else statement (or something with that effect)

Reply:
I have changed the structure of my addTargetBlankToLinks to always return the html whether it has been edit or not. Confirmed the errors you were getting previously are fixed on local.

Reply:
I would recommend using the regex
```js
/<a(?![^>]+target=)/g
```
as it'd be more specific than just target. As unlikely as it may be, there could be a random situations like the following where it'd misbehave with your current regex:
```html
<a target-value="true" href="foobar">
<a href="https://www.target.com">
```

Reply:
Added the new regex in latest commit